src/nccl_ofi_rdma.c

```diff
@@ -3818,24 +3835,68 @@ static int recv_close_deferred(nccl_net_ofi_recv_comm_t *recv_comm)
 	return ret;
 }
 
+static int alloc_rdma_flush_req(nccl_net_ofi_rdma_recv_comm_t *r_comm,
```

Comment:
`<object>_<verb>`. so `rdma_comm_alloc_frush_req()`

Reply:
Is this a new rule? many existing function names are like `alloc_rdma_<verb>_req`, examples :   `alloc_rdma_read_req`, `alloc_rdma_write_req`, `alloc_rdma_send_req`.

Reply:
it's been a poorly enforced rule.